forc/Cargo.toml

```
@@ -23,10 +23,10 @@ clap = { version = "3.1", features = ["cargo", "derive", "env"] }
 clap_complete = "3.1"
 forc-pkg = { version = "0.11.0", path = "../forc-pkg" }
 forc-util = { version = "0.11.0", path = "../forc-util" }
-fuel-asm = "0.3"
-fuel-gql-client = { version = "0.5", default-features = false }
-fuel-tx = "0.7"
-fuel-vm = "0.6"
+fuel-asm = "0.4"
+fuel-gql-client = { version = "0.6.2", default-features = false }
```

These should just be major.minor throughout, along with `fuel-core`.
```suggestion
fuel-gql-client = { version = "0.6", default-features = false }
```

---

I remember things didn't work without the `.2` 🤔 but I will try again once I'm through with all the bumps.

---

Yup seems to be okay now with just major.minor. Updated everywhere.